### ISSUE TITLE
Kind scripts broken?

### ISSUE THREAD

**Issue by johnSchnake** [aquasecurity/kube-bench#457]
I'm trying to run the commands given in the readme to run against a Kind cluster but it is failing.

My first instinct would be that something is just misconfigured, but since the deployment method is defined (Kind) and the cluster version is specified in the makefile (v1.15.3) I would expect the commands be configured properly.

Any advice?

The error in question is:
```
kube-bench-qlt7f kube-bench
kube-bench-qlt7f kube-bench need kubelet executable but none of the candidates are running
```

**Comment by robertojrojas** [comment]
@johnSchnake Thanks for the reporting this issue. 
Can you please provide the **kind** version you used so I can see if I reproduce your issue?
Also, if you add the parameter **"-v", "9"** to the container command, it'll display logs that will helps use see any errors.

We're also looking into changing the way we use **kind** as part of other internal changes we're making.

In the meantime, you can try the following steps:

- Install [kind v0.5.1](https://github.com/kubernetes-sigs/kind/releases/tag/v0.5.1) and latest **kubectl**
- `kind create cluster --image kindest/node:v1.15.3 --name kube-bench`

-  `export KUBECONFIG="$(kind get kubeconfig-path --name="kube-bench")"`
- `VERSION=<YOUR_VERSION> DOCKER_REGISTRY=<DOCKER_REPO> GO111MODULE=on make build-docker`
- `docker push <DOCKER_REPO>/kube-bench:<YOUR_VERSION>`
- modify the **job-node.yaml** file like this:
     

   1.  change the image to use your image version 
         `image: <DOCKER_REPO>/kube-bench:<YOUR_VERSION>`
    2. change the **container command** to look like this:
        `command: ["kube-bench","node", "--version", "1.13"]`
   3. comment out the following 
        `  #- name: usr-bin `
         ` #      mountPath: /usr/bin`

- `kubectl apply -f job-node.yaml`

- display job logs 
     `kubectl logs $(kubectl get pod | grep kube-bench-node | head -n 1 | awk '{print $1}') `

**Comment by robertojrojas** [comment]
@johnSchnake Thanks for your contribution!
Fixed by #459 
Closing.
